examples/msw/README.md

```diff
@@ -0,0 +1,25 @@
+# Remix + MSW
+
+This example demonstrates [MSW's][msw] usage with Remix to mock external APIs called from the Remix server during development.
```

Comment:
It's a nitpick but the word "external" may make an impression that MSW is only for third-party API mocking. I think you're safer saying just "to mock any HTTP calls" (I'd even omit API as it can mean a bunch of things depending on the context).